### ISSUE TITLE
Bug: perf.getEntriesByName is not a function

### ISSUE THREAD

**Issue by micahjsmith** [ballet/ballet-bot#26]
```
TypeError: perf.getEntriesByName is not a function
  File "/app/node_modules/marky/lib/marky.cjs.js", line 55, col 24, in Object.exports.stop
    var entries = perf.getEntriesByName(name);
  File "/app/node_modules/isomorphic-git/dist/for-node/isomorphic-git/index.js", line 2966, col 19, in null.<anonymous>
    marky.stop('percent').duration
  File "/app/node_modules/isomorphic-git/dist/for-node/isomorphic-git/index.js", line 2763, col 9, in listpack
    onData({
  File "/app/node_modules/isomorphic-git/dist/for-node/isomorphic-git/index.js", line 2950, col 5, in async Function.fromPack
    await listpack([pack], ({ data, type, reference, offset, num }) => {
  File "/app/node_modules/isomorphic-git/dist/for-node/isomorphic-git/index.js", line 7233, col 19, in async fetch
    const idx = await GitPackIndex.fromPack({
  File "/app/node_modules/isomorphic-git/dist/for-node/isomorphic-git/index.js", line 7671, col 42, in async Object.clone
    const { defaultBranch, fetchHead } = await fetch({
  File "/app/lib/github.js", line 17, col 3, in async Object.downloadRepo
    await git.clone({
  File "async /app/index.js", line 35, col 21, in null.<anonymous>
```

**Comment by micahjsmith** [comment]
Unable to reproduce locally on node 15

```
curl \
    -X POST \
    -H @headers.txt \
    -d @payload.json \
    http://localhost:3000
```

Response
```
14:37:01.196Z  INFO event: Responding to check_run (id=48087870-a94e-11eb-839c-1d382b88a3e7) (id=48087870-a94e-11eb-839c-1d382b88a3e7)
14:37:02.554Z  INFO event: Getting a check from branch: master (id=48087870-a94e-11eb-839c-1d382b88a3e7)
14:37:02.554Z  INFO event: On commit: Add a helpful message on errors loading data (id=48087870-a94e-11eb-839c-1d382b88a3e7)
14:37:02.717Z  INFO event: Not acting to prune features because not on master branch after merge commit (id=48087870-a94e-11eb-839c-1d382b88a3e7)
14:37:02.717Z  INFO event: Not acting to merge PR because not a PR (id=48087870-a94e-11eb-839c-1d382b88a3e7)
14:37:02.717Z  INFO event: Not acting to close PR because not a pull request (id=48087870-a94e-11eb-839c-1d382b88a3e7)
14:37:02.723Z  INFO http: POST / 200 - 1537.31 ms (id=480
```

**Comment by micahjsmith** [comment]
Able to reproduce locally by upgrading to node 16. Upgrading deps does not fix.

Can fix by downgrading to node 14 or 15. Not exactly sure which dep has a problem, presumably isomorphic-git which has v1 and v2 major releases by now. But a quick attempt to upgrade to v1 did not fix.

**Comment by micahjsmith** [comment]
Fixed in 5401da2974f174dce6bc3f9fa8e9c4927a73ec11
